index.test-d.ts

```diff
@@ -1,12 +1,12 @@
 import { AwilixContainer } from 'awilix'
 import fastify, { FastifyInstance } from "fastify";
-import { diContainer, FastifyAwilixOptions, fastifyAwilixPlugin } from './'
+import { diContainer, FastifyAwilixOptions, fastifyAwilixPlugin, Cradle } from './'
 import { expectAssignable, expectType } from 'tsd'
 
 expectAssignable<FastifyAwilixOptions>({})
 expectAssignable<FastifyAwilixOptions>({ disposeOnClose: false})
 expectAssignable<FastifyAwilixOptions>({ disposeOnResponse: false})
-expectType<AwilixContainer>(diContainer)
+expectType<AwilixContainer<Cradle>>(diContainer)
```

Comment:
Could you add interface extension in this file and actually assert that extension takes effect both for this and for requestcradle?

Reply:
Great suggestion 👌 , I've also used this file to provide simple usage example for users